examples/performance/stress-test/app.js

```diff
@@ -0,0 +1,293 @@
+import GL from '@luma.gl/constants';
+import {
+  AnimationLoop,
+  setParameters,
+  clear,
+  Texture2D,
+  Buffer,
+  Model,
+  CubeGeometry
+} from '@luma.gl/core';
+import {Matrix4, radians} from 'math.gl';
+/* eslint-disable spaced-comment, max-depth */
```

Comment:
Nits:
- Usually place linter "imports". i.e.  `/* global .. */` above the import section.
- When possible, please use `// eslint-disable-line rule` or `// eslint-disable-next-line rule` to minimize the amount of code affected by the disable. May not be practical for the `spaced-comment` but should be used for the `max-depth`.